google-pub/c/mqtt_ciotc.c

```diff
@@ -324,11 +324,28 @@ int Publish(char* payload, int payload_size) {
 }
 // [END iot_mqtt_publish]
 
+// check if running as root
+void checkRoot(void)
+{
+  int euid = geteuid();
```

Comment:
Indentation different here.

Reply:
This is aligned with indentation style of the source code in this sample